curt.py

```diff
@@ -1371,13 +1368,23 @@ def print_task_stats(tasks):
 					report_calls(task.irqs, irq_name, process.irqs, system.irqs)
 					print
 
-		process.output_header()
-		print
-		print "\t[     ALL] %-20s ALL" % (""),
-		process_cpus_sum.output()
-		process.output_migrations()
-		print
-		print
+				# Adding per cpu per process info
+				process.output_header()
+				print
+				############################################
+				print 'Per CPU per process--START'
+				for cpu in process.cpus:
+					print "\t[%8s] %-20s %3u" % (pid, comm, cpu),
```

Comment:
I haven't thought about this deeply, but this shows up as `[ unknown] some-command #`. The previous use of the field with brackets `[]` is for TIDs, so we should be consistent.  I think we should hardcod "ALL" here, since this is indeed for all tasks in this process.